src/components/VContentReport/VContentReportForm.vue

```diff
@@ -93,10 +93,11 @@
           <VButton
             v-else
             key="non-dmca"
+            type="submit"
             :disabled="isSubmitDisabled"
             :focusable-when-disabled="true"
             variant="secondary"
-            @click="handleSubmit"
+            :value="$t('media-details.content-report.form.submit')"
```

Comment:
Fun fact: Playwright's selectors for text `page.locate('text="Submit"'), for example, use the `value` when applied to submit buttons, not the actual button text contents.

Reply:
TIL! I guess we should add `value` attribute to all the submit buttons? I wonder if it uses `value` for all buttons...